python/docs/source/conf.py

```diff
@@ -0,0 +1,73 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import toml
+
+sys.path.insert(0, os.path.abspath("../deltalake/"))
+
+
+def get_release_version() -> str:
+    """
+    Get the release version from the Cargo.toml file
+
+    :return:
+    """
+    cargo_content = toml.load("../../Cargo.toml")
+    return cargo_content["package"]["version"]
+
+
+# -- Project information -----------------------------------------------------
+
+project = "delta-rs"
+copyright = ""
+author = "Florian Valeye"
```

Comment:
```suggestion
copyright = "2021 delta-rs contributors"
author = "delta-rs contributors"
```

I think it's best to leave this general and vague, not that the Sphinx configuration has any legal basis anyways :smile_cat: